.github/workflows/go.yml

```diff
@@ -0,0 +1,44 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This should also run on Pull requests?

Reply:
with `pull_request` , It will run

Reply:
Don't you just need this: https://github.com/zalando-incubator/kube-ingress-aws-controller/blob/3bf44bedcfefa4d7a220dffeaa5e749933280c4d/.github/workflows/ci.yaml#L2

With `branches: [master]` I think it's limited to only run when merged to master? I don't see that it's running for this PR for example.

Reply:
`branches: [ master ]` runs on when a pull request created against the `zalando:master `. Creator's branch is not important. There aren't any branch like `dev` that is used before merging to the master.

> I don't see that it's running for this PR for example.

This action will run for next pull requests after it is merged.

Example: https://github.com/erdaltsksn/gh-label/pull/2/checks

I can remove them if you want more permissive configurations.

Reply:
Thanks for clarifying. That sounds ok for me as-is. Can you sign-off your commits then we're good to go :)

Reply:
✅